cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -45,11 +45,9 @@ In {odo-title}, services are provisioned from the OpenShift Service Catalog and
 |====
 |Node.js|Java
 |`centos/nodejs-8-centos7`|`redhat-openjdk-18/openjdk18-openshift`
-|`rhoar-nodejs/nodejs-8`|`openjdk/openjdk-11-rhel8`
-|`rhoar-nodejs/nodejs-10`|`openjdk/openjdk-11-rhel7`
+|`rhscl/nodejs-10-rhel7`|`openjdk/openjdk-11-rhel8`
+|`rhscl/nodejs-12-rhel7`|`openjdk/openjdk-11-rhel7`
```

Comment:
We are still using the name space ```rhoar-nodejs``` in our actual odo code - https://github.com/openshift/odo/blob/425d77670264efc502e02aed28a5665546bfefd8/pkg/catalog/catalog.go#L282-L290. Can you please share the supported reference to this change.

@kadel If this is the case then we need to update the image registry in our odo code base too. PTAL

Reply:
This was the blog post that sort of talks about the 1 node.js from redhat,  https://developers.redhat.com/blog/2020/01/07/red-hat-support-for-node-js/

I noticed in that code that there are also references to bucharestgold/node images.  We've actually changed the org name to nodeshift,  but we've actually replaced those community images with the ubi based ones from rhscl.   looks like that should be another PR :)

Reply:
Also,  related to my last comment,  Node 8 went EOL at the end of 2019,  so those references should be removed and Node 12 can be added